EDU-2644: Expands graceful failover to mention retry period (#2909)
temporalio/documentation: docs/production-deployment/cloud/multi-region.mdx

The Namespace experiences a short period of unavailability, defaulting to 10 seconds.

During this period, existing Workflows stop progress.

> Temporal Cloud will return a "Service unavailable error", which is retried by SDKs.

Author: ```suggestion
Temporal Cloud will return a "Service unavailable error", which will be retried by SDKs.
```

> Temporal Cloud will return a "Service unavailable error", which will be retried by SDKs.

Reviewer: ```suggestion
Temporal Cloud returns a "Service unavailable error", which is retried by SDKs.
```

I **think** the style guidance says to use present tense in this situation.

Reviewer: If you adopt here, suggest adopting for the other sentence below.

#### Hybrid failover mode {#hybrid-failover}

While graceful failovers are preferred for consistency, they aren’t always practical.

> Temporal Cloud’s hybrid failover mode (the default mode) limits a single graceful failover attempt to 10 seconds or less.

Author: ```suggestion
Temporal Cloud’s hybrid failover mode (the default mode) limits an initial graceful failover attempt to 10 seconds or less.
```

While graceful failovers are preferred for consistency, they aren’t always practical.
Temporal Cloud’s hybrid failover mode (the default mode) limits an initial graceful failover attempt to 10 seconds or less.
During this period, existing Workflows stop progress.

> Temporal Cloud will return a "Service unavailable error", which will be retried by the SDKs.

Author: ```suggestion
Temporal Cloud returns a "Service unavailable error", which is retried by SDKs.
```
